src/observable/collected.jl

```diff
@@ -94,7 +100,11 @@ function next_received!(wrapper::CollectLatestObservableWrapper, data, index::Ca
     @inbounds wrapper.ustatus[index] = true
     if all(wrapper.vstatus) && !all(wrapper.cstatus)
         unsafe_copyto!(wrapper.vstatus, 1, wrapper.cstatus, 1, length(wrapper.vstatus))
-        next!(wrapper.actor, wrapper.mappingFn(wrapper.storage))
+        value = wrapper.mappingFn(wrapper.storage)
```

Comment:
Might be better to parameterize `CollectLatestObservableWrapper` here? I think it will compile away the call to `isnothing` if {C} is nothing already

Reply:
It is parametrized with `C` already